seo/speed.md

```diff
@@ -0,0 +1,47 @@
+# Front end speed
```

Comment:
Can we update the current culture doc page with any extra info: https://github.com/holidayextras/culture/blob/master/front-end-performance.md As this advice applies to all of our projects.